### ISSUE TITLE
Components

### ISSUE THREAD

**Issue by truenorth** [truenorth/ember-material-lite#1]
- [ ] Badges
- [ ] Buttons
- [ ] Cards
- [ ] Layout
- [ ] Loading
- [ ] Menus
- [ ] Sliders
- [ ] Toggles
- [ ] Tables
- [ ] Text Fields
- [ ] Tooltips

**Comment by sunil-shrestha** [comment]
It would be great to have side navigation with drawer also. Any plan for this feature or is it included with Navbar ?

**Comment by truenorth** [comment]
@sunil-shrestha it's already included with the navbar

**Comment by sunil-shrestha** [comment]
Nice. Now i can see it in documentation also. Sample codes are not appearing though.

**Comment by truenorth** [comment]
@sunil-shrestha example code should be showing up again now. Thanks for your patience

**Comment by rynam0** [comment]
This is coming along quite nicely!  Nice work.

**Comment by truenorth** [comment]
We now have a decent implementation of all components. I'll start to back-fill tests, and ensure we have a consistent API surface whenever possible

**Comment by rynam0** [comment]
Any intention on adding checkbox list like in ember-cli-materialize, Mike?

**Comment by sunil-shrestha** [comment]
First of all, big thanks to @truenorth  for bringing Material-design-lite to ember community. 

I have been testing with Nav components and so far looks great. Now i come to a case where the drawer nav-items are different from the main nav items. If i am not wrong, we now have same nav items in side nav as on the main nav. Do you have any plan to include this kind of use case or is it possible already?

**Comment by truenorth** [comment]
Until we have some more sophisticated content projection options, there's no way to do this without making use of private APIs (and thus introducing risk to consuming apps). 

My advice is to extend from the existing nav component, but supply your own template.

**Comment by sunil-shrestha** [comment]
Thanks, the latest release seem to provide enough flexibility for Nav now.

**Comment by truenorth** [comment]
Yep, I was able to use [ember-composability](https://github.com/truenorth/ember-composability) to make this work.

**Comment by rohanthacker** [comment]
Do you currently handle layouts when using this addon? Especially across different views.
The nav-bar overlaps content and when fixedDrawer is enabled it overlaps the main content

**Comment by mike-north** [comment]
@rohanthacker not sure exactly what you're asking. Can you elaborate?

**Comment by rohanthacker** [comment]
@mike-north My apologies. I want to know how you handle layout when using this addon.?  Because currently the nav-bar overlaps the content of the page. Also when  fixedDrawer is enable, it too overlaps the content.

Screenshot of nav overlapping content
[Imgur](http://i.imgur.com/arGVfcZ.png)
 
Screenshot of drawer overlapping content
 [Imgur](http://i.imgur.com/V7BtaKy.png)

**Comment by mike-north** [comment]
@rohanthacker This should be handled in the same way that Material Design Lite handles it http://www.getmdl.io/components/index.html#layout-section/layout

**Comment by remkoboschker** [comment]
Are there any plans to support the new components dialogs, lists and snackbar? I understand that pull request are welcome.
